content/docs/06-integrations/00-aws-efs.md

```diff
@@ -163,24 +324,35 @@ Events:
 Mounted By:  <none>
 
 ```
+Possible Troubleshooting info to add:
 
-# References
-
-See the following information on all AWS EFS parameters:
+If you get errors in your pods when mounting your Amazon EFS volume in your Amazon EKS cluster, verify you have the following:
 
-- https://github.com/kubernetes-sigs/aws-efs-csi-driver
+- An Amazon EFS file system created with a mount target in each of the worker node subnets.
+- A valid EFS storage class (from the GitHub website) definition using the efs.csi.aws.com provisioner.
+- A valid PersistentVolumeClaim (PVC) definition and PersistentVolume definition. This isn't needed if you're using dynamic provisioning (from the GitHub website).
+- The Amazon EFS CSI driver installed in the cluster.
 
-- https://aws.amazon.com/efs/
 
-- https://github.com/aws-samples/amazon-efs-tutorial
+# Terraform
 
+```
+data "spectrocloud_registry" "container_registry" {
+  name = "Public Repo"
+}
 
+data "spectrocloud_pack_simple" "csi-aws-efs" {
+  name    = "csi-aws-efs"
+  version = "1.4.0"
+  type = "helm"
+  registry_uid = data.spectrocloud_registry.container_registry.id
+}
```

Comment:
```suggestion
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}

data "spectrocloud_pack_simple" "csi-aws-efs" {
  name    = "csi-aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Reply:
@ritawatson, so the current Terraform code is if you are using this Pack as an add-on to a cluster profile.

When using this Pack as a base layer, you need the following terraform code.

```
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}
data "spectrocloud_pack_simple" "aws-efs" {
  name    = "aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Reply:
Pink is system app (storage layer), orange is addon
![CleanShot 2023-01-30 at 19 35 56@2x](https://user-images.githubusercontent.com/29551334/215648255-d33867a0-e6e9-4f9c-b9be-98e5638c314c.png)

Reply:
I asked in a thread that I tagged you on if this is the plan.